modules/ROOT/pages/java-version-support.adoc

```diff
@@ -0,0 +1,63 @@
+= Java Version Support
+
+include::partial$specify-java-compatibility.adoc[]
+
+Communicating the Java compatibility of your custom module also ensures:
+
+* Safe upgrades
++
+Users can determine whether their Mule applications can safely upgrade to Java 17, ensuring a smooth transition and avoiding potential issues.
+
+* Server health
++
+Certain module versions might cause problems or resource leaks when deployed on Java 17, so identifying and avoiding these versions helps maintain overall server health.
```

Comment:
where did you get this statement from? Maybe it's out of context, but this is not true (at least in the way it's worded here)